.github/workflows/test.yaml

```
@@ -34,3 +34,18 @@ jobs:
         run: yarn test
         env:
           CI: true
+
+  inclusive-naming-check:
+    name: Inclusive naming check
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+
+      - name: woke
```

More descriptive step name

```suggestion
      - name: Check inclusive naming
```